test/bdd/pkg/edv/edv_steps.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package edv
+
+import (
+	"fmt"
+
+	"github.com/trustbloc/edv/pkg/client/edv"
+	"github.com/trustbloc/edv/pkg/restapi/edv/operation"
+	"github.com/trustbloc/edv/test/bdd/pkg/context"
+
+	"github.com/DATA-DOG/godog"
+)
+
+// Steps is steps for EDV BDD tests
+type Steps struct {
+	bddContext *context.BDDContext
+}
+
+// NewSteps returns new agent from client SDK
```

Comment:
`new agent from client SDK`?

Reply:
Oops. I also found a few other places with "agent" erroneously included in comments. Fixed.